web/src/app/app.component.html

```diff
@@ -23,8 +23,8 @@
     <taskana-shared-nav-bar></taskana-shared-nav-bar>
     <div (window:resize)="onResize()">
       <div class="taskana-main">
-        <div class="taskana-main__progress-bar">
-          <mat-progress-bar [hidden]="!requestInProgress" mode="indeterminate"></mat-progress-bar>
+        <div class="taskana-main__progress-bar" [hidden]="requestNotInProgress | async">
+          <mat-progress-bar mode="indeterminate"></mat-progress-bar>
```

Comment:
With this change, the space for the progress bar is deleted as soon as the progress bar disappears. Consequently, the whole view shifts a little bit down and up. Please find another solution.

Reply:
Additionally, the progress bar doesn't disappear after starting the backend and opening the app as soon as the backend is ready.

Reply:
Replaced with an *ngIf, as [hidden] did not appear to work

Reply:
The new solution causes `ExpressionChangedAfterItHasBeenCheckedError ` every time the progress bar is shown.